torch_optimizer/yogi.py

```diff
@@ -9,37 +10,38 @@
 
 
 class Yogi(Optimizer):
-    r"""Implements Yogi optimization algorithm.
-
-    It has been proposed in `Adaptive Methods for Nonconvex Optimization`__.
-
+    r"""Implements Yogi Optimizer Algorithm.
+    It has been proposed in `Adaptive methods for Nonconvex Optimization`_.
     Arguments:
-        params: iterable of parameters to optimize or dicts defining
+        params (iterable): iterable of parameters to optimize or dicts defining
```

Comment:
No need to add types in doc strings, class constructor is fully types with `mypy` annotations docs builder looking them up there.